utility/src/main/java/org/oppia/android/util/image/bitmap/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+"""
+General purpose utilities for bitmap images.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "bitmap",
+    srcs = [
+        "BitmapBlurTransformation.kt",
+        "BitmapBlurrer.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
It seems like this library is only used by drawable, could you minimize the visibility to that directory?

Reply:
You can give a specific path to visibility. It will be an ad hoc visibility. Which is fine if it's only going to be used from one package.

Reply:
Ok you mean if it's used only in in image package use:
visibility = ["//utility/src/java/main/org/oppia/android/util/image"].

Did I get it right?

Reply:
Yes, exactly. This will ensure that we developers are not using libraries that are not meant to be used outside of a certain context.

Reply:
Thanks @fsharpasharp learnt something new so I'll change that visibility of others as well

Reply:
Done